internal/file/proto_set_provider.go

```diff
@@ -195,6 +218,17 @@ func (c *protoSetProvider) walkAndGetAllProtoFiles(absWorkDirPath string, absDir
 		timedOut       bool
 	)
 	allExcludes := make(map[string]struct{})
+	// if we have a configDirPath and configData, we compute the exclude prefixes once
+	// from this dirPath and data, and do not do it again in the below walk function
+	if c.configDirPath != "" {
```

Comment:
We can update this to `c.configData != ""`

Reply:
Done